.github/workflows/build.yml

```diff
@@ -66,14 +66,14 @@ jobs:
             -
                 name: Prepare test application
                 run: |
-                    (cd src/Bundle/test && app/console doctrine:database:create)
-                    (cd src/Bundle/test && app/console doctrine:schema:create)
+                    (cd src/Bundle/test && bin/console doctrine:database:create)
+                    (cd src/Bundle/test && bin/console doctrine:schema:create)
 
-            -
-                name: Run analysis
-                run: |
-                    composer analyse
-                    (cd src/Component && composer validate --strict)
+#            -
```

Comment:
@pamil It has to be solved.
This error occurs https://github.com/symplify/symplify/issues/2873
and this error occurs https://github.com/vimeo/psalm/issues/4961